### PR TITLE
Adding a CN-docs to support Chinese language support

### PR DESCRIPTION
**Pull request by Mark-InCrypto**
Add a CN-docs to support Chinese language

**Description**
Adding a CN-docs to support Chinese language support so can be more widely open to Chinese users.
Copy the original docs-folder firstly and put CN translation one-by-one.
First translation is for the getting-started=>以太坊 Optimistic 入门指南
And the  getting-started.md remains unchanged.

**Personal introduction**
Deep using crypto products, DeFi platforms, multiple L1 chains, multiple L2 channels, being crypto project Chinese community manager for several years, lots experience of crypto translation, willing to make optimism voice louder by providing CN translation content and spread to more users.

### CODE REVIEW

**Comment by platocrat** [comment]
Here's some comments from @Mark-InCrypto via Discord DMs.

Screen Shot 2021-11-16 at 10 38 36 AM

File: src/CN-docs/users/以太坊 Optimistic 入门指南.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
+---
+标题: 以太坊 Optimistic 入门指南
+语言: 中文
+---
+
+# {{ $frontmatter.title }}
+
+::: 提示 OVM 2.0 发布日期
```
Comment (LeifuChen) [inline src/CN-docs/users/以太坊 Optimistic 入门指南.md:8]: ::: tip 是[VuePress的标记](https://v1.vuepress.vuejs.org/guide/markdown.html#custom-containers)，可以不用翻译。

File: src/CN-docs/users/以太坊 Optimistic 入门指南.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+OVM 2.0 已经在 Kovan 测试网络上发布。
+我们预计将在 11 月 11 日将其部署到以太网 Optimistic 网络。
```
Comment (LeifuChen) [inline src/CN-docs/users/以太坊 Optimistic 入门指南.md:10]: 后半句建议改为 ```将其部署到以太坊 Optimistic 生产环境```。

File: src/CN-docs/users/以太坊 Optimistic 入门指南.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+:::
+
+## 为什么使用以太坊 Optimistic 网络?
+
+以太坊 Optimistic 网络允许您发送交易，类似于以太坊，但具有两个重要优势：
+
+* **近乎即时的交易结果反馈。** 您几乎可以立即知道您的交易是否完成。
+* **更低的交易手续费。** 通常在 L1 成本的 1% 到 10% 之间。
+  [关于当前的 gas 费用和一些示例交易的成本，请参见此处。](https://public-grafana.optimism.io/d/9hkhMxn7z/public-dashboard?orgId=1&refresh=5m).
```
Comment (LeifuChen) [inline src/CN-docs/users/以太坊 Optimistic 入门指南.md:19]: 链接后面的```.```可以去掉，因为```。```已经包含在```[...请参见此处。]```里面了。

File: src/CN-docs/users/以太坊 Optimistic 入门指南.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+* **去中心化。** 所有交易都发布到L1 以太坊，继承了以太坊强大的安全保障
+
+## 存放资产
+
+在 Optimistic 以太坊上使用资产（Eth、ERC-20 代币等）之前，您需要使用 [跨链桥](https://www.optimism.io/apps/dapps). 对于存入 ETH，您可以使用 [我们自己的跨链桥](https://gateway.optimism.io/).
```
Comment (LeifuChen) [inline src/CN-docs/users/以太坊 Optimistic 入门指南.md:24]: 这里的```.```建议改成```。```。
类似的问题下面也有，建议都统一改一下。

File: src/CN-docs/users/以太坊 Optimistic 入门指南.md [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+
+1. 如果您没有 ETH，可以通过中心化交易所购买一些，例如 
+   [Coinbase](https://www.coinbase.com/signup), [Gemini](https://exchange.gemini.com/register),
```
Comment (LeifuChen) [inline src/CN-docs/users/以太坊 Optimistic 入门指南.md:27]: - ```，```建议改成```、```。
- ```或者```前面不需要再加```，```。

File: src/CN-docs/users/以太坊 Optimistic 入门指南.md [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+     <img src="../../assets/docs/users/getting-started/deposit-form-b.png" alt="Deposit form" width="40%" style="float:left">
+     </div>
+
+   * 选定代币为 **ETH**.
+
+     <div style="display:inline-block">
+     <img src="../../assets/docs/users/getting-started/deposit-form-c.png" alt="Deposit form" width="40%" style="float:left">
+     </div>
+
+   * 数额是可行的。可以单击MAX查看您可以存入的最大数额（去除转账手续费以后），如果相关，请输入低于MAX显示的数额。
```
Comment (LeifuChen) [inline src/CN-docs/users/以太坊 Optimistic 入门指南.md:52]: 这句话的英文感觉就不是很顺。我建议放弃逐字翻译，可意译为```请确保此处输入的金额小于可存入的最大金额，可以点击MAX查看您当前可存入的最大金额（因为考虑转账手续费，所以会略小于钱包余额）。```

File: src/CN-docs/users/以太坊 Optimistic 入门指南.md [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+      <img src="../../assets/docs/users/getting-started/swap-form-a.png" alt="Swap form" width="40%" style="float:left"> 
+      </div>
+
+   B. 您想要交易获得的代币
+
+      <div style="display:inline-block">
+      <img src="../../assets/docs/users/getting-started/swap-form-b.png" alt="Swap form" width="40%" style="float:left">
+      </div>
+
+   C. 输入任意代币的数额
```
Comment (LeifuChen) [inline src/CN-docs/users/以太坊 Optimistic 入门指南.md:94]: 这里也建议意译：```输入您愿意支付的代币金额或者希望交易取得的代币金额。```

Comment (Mark-InCrypto) [reply src/CN-docs/users/以太坊 Optimistic 入门指南.md:94]: 感谢！
金额用数额感觉更直观、明了（个人理解）。

调整为：输入您愿意支付的代币数额或者在下行输入希望通过交易获得的代币数额。

File: src/CN-docs/users/以太坊 Optimistic 入门指南.md (hunk 6 above)
Comment (Mark-InCrypto) [reply src/CN-docs/users/以太坊 Optimistic 入门指南.md:52]: 感谢！

感觉金额用数额更妥当。

考虑了下，调整为如下：
请确保此处输入的ETH数额小于钱包余额，同时您可以点击MAX查看您当前可存入的最大数额（由于考虑转账手续费，所以MAX数额会略小于钱包余额）。

File: src/CN-docs/users/以太坊 Optimistic 入门指南.md (hunk 5 above)
Comment (Mark-InCrypto) [reply src/CN-docs/users/以太坊 Optimistic 入门指南.md:27]: 感谢🙏！

很细心！

File: src/CN-docs/users/以太坊 Optimistic 入门指南.md (hunk 4 above)
Comment (Mark-InCrypto) [reply src/CN-docs/users/以太坊 Optimistic 入门指南.md:24]: 感谢🙏！

很细心！

File: src/CN-docs/users/以太坊 Optimistic 入门指南.md (hunk 3 above)
Comment (Mark-InCrypto) [reply src/CN-docs/users/以太坊 Optimistic 入门指南.md:19]: 感谢🙏！

很细心！

File: src/CN-docs/users/以太坊 Optimistic 入门指南.md (hunk 2 above)
Comment (Mark-InCrypto) [reply src/CN-docs/users/以太坊 Optimistic 入门指南.md:10]: 感谢！
已修改。

File: src/CN-docs/users/以太坊 Optimistic 入门指南.md (hunk 1 above)
Comment (Mark-InCrypto) [reply src/CN-docs/users/以太坊 Optimistic 入门指南.md:8]: 感谢！

学习到了！

File: src/CN-docs/users/以太坊 Optimistic 入门指南.md (hunk 6 above)
Comment (LeifuChen) [reply src/CN-docs/users/以太坊 Optimistic 入门指南.md]: 对，你这样翻译出来的意思比我修改得更明确一些。

File: src/CN-docs/users/以太坊 Optimistic 入门指南.md (hunk 7 above)
Comment (LeifuChen) [reply src/CN-docs/users/以太坊 Optimistic 入门指南.md]: 我觉得这样改得很好！

File: src/CN-docs/users/OVM 2.0 中的交易费用.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+::: tip
+OVM 2.0 页面 此页面指的是OVM 2.0 更新后以太坊 Optimistic 网络的**最新**状态。:::
+
+## 费用简述
+
+在大多数情况下，以太坊 Optimistic 网络上的转账手续费要明显低于以太坊主网上的费用。
+每一笔以太坊 Optimistic 网络手续费都包含两个成本:
+
+您支付两项费用的总和：您的 L2（执行）费用和您的 L1 （安全）费用。在较高级别上，
```
Comment (LeifuChen) [inline src/CN-docs/users/OVM 2.0 中的交易费用.md:22]: - ```安全```改成```安保```(如果修改的话，本节后面还有一处用到```L1 安全手续费```的，可以一起修改)
- ```在较高级别上```改成```宏观来说```
会不会理解上更直观一些？

File: src/CN-docs/users/OVM 2.0 中的交易费用.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+L2 费用是在 L2 中执行您的交易的成本，而 L1 费用是将您的交易提交至 L1 （在汇总批次中） 对预估成本。
```
Comment (LeifuChen) [inline src/CN-docs/users/OVM 2.0 中的交易费用.md:23]: ```对预估成本```改成 ```的预估成本```

**Comment by smartcontracts** [comment]
This is pretty amazing, thank you to everyone contributing here :-). We'll get some people from the OP team to take a look at this soon as well!!

File: src/CN-docs/users/小狐狸钱包.md [hunk 10]
```diff
@@ -0,0 +1,43 @@
+---
+标题: 通过小狐狸钱包链接以太坊Optimistic 网络
+语言: 中文
+---
+
+::: tip
```
[the remark this answers is not in the document]
Comment (Mark-InCrypto) [reply src/CN-docs/users/小狐狸钱包.md:6]: 感谢！

File: src/CN-docs/users/OVM 2.0 中的交易费用.md [hunk 11]
```diff
@@ -0,0 +1,37 @@
+---
+标题: OVM 2.0 中的交易费用
```
[the remark this answers is not in the document]
Comment (Mark-InCrypto) [reply src/CN-docs/users/OVM 2.0 中的交易费用.md:2]: 已更新！

File: src/CN-docs/users/OVM 2.0 中的交易费用.md (hunk 8 above)
Comment (Mark-InCrypto) [reply src/CN-docs/users/OVM 2.0 中的交易费用.md:22]: 思考了下，对安保再写详细一些，调整为L1（安全保障）费用。  L1 安全保障手续费。
第二条已修改。

File: src/CN-docs/users/OVM 2.0 中的交易费用.md (hunk 9 above)
Comment (Mark-InCrypto) [reply src/CN-docs/users/OVM 2.0 中的交易费用.md:23]: 因为电脑输入法的问题，没注意一些的的错误，都已经修改好了。感谢！

**Comment by Mark-InCrypto** [comment]
> This is pretty amazing, thank you to everyone contributing here :-). We'll get some people from the OP team to take a look at this soon as well!!

**Comment by Mark-InCrypto** [comment]
> This is pretty amazing, thank you to everyone contributing here :-). We'll get some people from the OP team to take a look at this soon as well!!

During the stay in Discord and noticing some misunderstanding in community,  thanks to @platocrat for setting up this pull request and thanks to @ LeifuChen for the translation review.

**Review by annieke — COMMENTED** [review]
好棒！

skimmed this and it looks great, @qbzzt @elenadimitrova , does the `CN-docs` repo structure make sense here? content-wise i think it is good to merge 💯

**Comment by qbzzt** [comment]
> skimmed this and it looks great, @qbzzt @elenadimitrova , does the CN-docs repo structure make sense here? content-wise i think it is good to merge 💯

The translations are for the user docs. Before we decide where to place them exactly we need to resolve whether the user docs stay in Vuepress or whether they go to Intercom. Both platforms have multilingual support, but in different ways.

**Comment by smartcontracts** [comment]
Announcement regarding our plan for the next few weeks, including information about translations going forward: #283

**Comment by qbzzt** [comment]
Sorry for taking so long, we finally settled on Zendesk. It's not quite ready yet, but the Chinese translation is at https://optimismhelp.zendesk.com/hc/zh-cn . Could you please tell me if I have any mistakes? I probably do, I don't know Chinese.

**Comment by Mark-InCrypto** [comment]
> https://optimismhelp.zendesk.com/hc/zh-cn

Just check the content, seems all the pics are down from my side, please have a look.

**Comment by LeifuChen** [comment]
> > https://optimismhelp.zendesk.com/hc/zh-cn
> 
> Just check the content, seems all the pics are down from my side, please have a look. If anything I can support/help, just let me know. My discord: Mark Crypto(🐇,🎩)_/#5798 (currently a support-nerd in Optimism server).

@qbzzt I can confirm that the pics are down because all the links of pics are using absolute path like ```https://localhost:8080/assets/img/``` Could you please re-check the settings?

**Comment by qbzzt** [comment]
I fixed it - I apologize, it worked on my machine, but that's because I run the Vuepress site locally. Now it should work without it.

**Comment by qbzzt** [comment]
We reorganized the user docs, and changed some category names. Can you please go to https://optimismhelp.zendesk.com/hc/zh-cn and tell me what they need to be?

**Comment by Mark-InCrypto** [comment]
> 

Thanks for the update, I will go through the docs and feedback to you later.

**Comment by LeifuChen** [comment]
@qbzzt Let's link this pull request with issue #222

**Comment by qbzzt** [comment]
We now have [CrowdIn](https://crowdin.com/project/optimism-help-center) for translations. I think I put all the translations in the PR into ZenDesk and it imported them, but I could be wrong. I'd love to have you use that.

谢谢

**Comment by Mark-InCrypto** [comment]
> We now have [CrowdIn](https://crowdin.com/project/optimism-help-center) for translations. I think I put all the translations in the PR into ZenDesk and it imported them, but I could be wrong. I'd love to have you use that.
> 
> 谢谢

I am starting use Crowdln platform for the currently translation review including the details you mentioned above and at the same time for starting the rest content translation tasks, for your reference! 

Mark Crypto(🐇,🎩)_/#5798

File: src/CN-docs/users/从以太坊 Optimistic 网络提款.md [hunk 12]
```diff
@@ -0,0 +1,137 @@
+---
+标题: 从以太坊 Optimistic 网络提款
+语言: CN-中文
+---
+
+# {{ $frontmatter.title }}
+
+::: tip 
+我的资金在那里？
+如果您使用Optimism 网官进行提款并且不确定您的提款发生了什么，请
```
Comment (imtipi) [inline src/CN-docs/users/从以太坊 Optimistic 网络提款.md:10]: 找到一个小错别字（

Comment (Mark-InCrypto) [reply src/CN-docs/users/从以太坊 Optimistic 网络提款.md:10]: > 找到一个小错别字（

https://crowdin.com/project/optimism-help-center/zh-CN

目前已至新的平台，可以直接提供翻译的反馈和更新了！

**Comment by Mark-InCrypto** [comment]
> We now have [CrowdIn](https://crowdin.com/project/optimism-help-center) for translations. I think I put all the translations in the PR into ZenDesk and it imported them, but I could be wrong. I'd love to have you use that.
> 
> 谢谢

I would suggest team can share this information to community, I am pretty sure community would like to contribute to the translation.

Currently the Chinese community have almost finished 100% the content.

**Comment by qbzzt** [comment]
Now that we have Chinese on CrowdIn (BTW, a huge **thank you** to @Mark-InCrypto and @imtipi), is it OK to close this PR and the issue connected to it?

**Comment by Mark-InCrypto** [comment]
> Now that we have Chinese on CrowdIn (BTW, a huge **thank you** to @Mark-InCrypto and @imtipi), is it OK to close this PR and the issue connected to it?

I agree.
Thanks for the support and focus on the users docs for multiple languages.
It is what I mean that the Pull Request can close now.
Any issues needed, just feel free to me as MarkCrypto(🐇,🎩)_/#5798 and I am currently also a support-nerd in Optimism discord. 

Mark.

**Comment by silentwars** [comment]
你好可以和你取得联系吗？懂一点点技术的我，请教一下该站点是怎么建立起来的，非常感谢。

**Comment by LeifuChen** [comment]
> 你好可以和你取得联系吗？懂一点点技术的我，请教一下该站点是怎么建立起来的，非常感谢。

这个好像不需要啥技术吧，你去zendesk的网站看一下怎么建立帮助文档？
https://support.zendesk.com/hc/en-us/articles/4408846795674-Getting-started-with-Guide-for-your-help-center-Setting-up

**Pull request closed without merging**